### PR TITLE
Added a events file as suggested to me under #3626 PR

### PR DESCRIPTION
**Pull request by abhay-raj19**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3650"></a>

### CODE REVIEW

File: data/abhay-raj19/2022-12-10-Google-Devfest'22.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "isVirtual": false,
+  "isInPerson": true,
+  "color": "green",
+  "name": "Google Devfest kanpur",
+  "description": "In this devfest,We had met many young Devlopers enlightining there carrer in Emerging Tech Domains,got hands into those teck-Stack,got an awesome Experience",
```
Comment (Parsifal-M) [inline data/abhay-raj19/2022-12-10-Google-Devfest'22.json:6]: Hope you don't mind the below:

`"In this devfest,We had met many young Devlopers enlightining there carrer in Emerging Tech Domains,got hands into those teck-Stack,got an awesome Experience"`

to:

`"At this devfest, We met many young Devlopers enlightining their career in Emerging Tech Domains, got hands-on into those tech stacks, and had an awesome experience"`

reads a lot better :smile:

**Review by Parsifal-M — COMMENTED** [review]
Some small nits, I hope you don't mind but it reads a lot better :smiley:

**Review by Parsifal-M — DISMISSED** [review]
LGTM!

File: data/abhay-raj19/2022-12-10-Google-Devfest'22.json (hunk 1 above)
Comment (abhay-raj19) [reply data/abhay-raj19/2022-12-10-Google-Devfest'22.json]: I had done the necessary changes.👍 pls see to it.

**Pull request closed without merging**